lib/beaker/host/pswindows/file.rb

```diff
@@ -2,17 +2,29 @@ module PSWindows::File
   include Beaker::CommandFactory
 
   def tmpfile(name)
-    execute("echo C:\\Windows\\Temp\\#{name}.%RANDOM%")
+    execute("echo #{system_temp_path}\\#{name}.%RANDOM%")
```

Comment:
If we're going to run back to the host for this through `execute`, better to just use PowerShell IMHO.

```powershell
[IO.Path]::GetTempFileName()
```